packages/ui/src/molecules/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,65 @@
+import type { ReactNode } from 'react'
+import React, { useRef, useMemo, useState, useEffect } from 'react'
+
+import DropdownContext from './contexts/DropdownContext'
+
+export type DropdownProps = {
+  children: ReactNode
+  onDismiss?(): void
+  isOpen?: boolean
+  id?: string
+}
+
+const Dropdown = ({
+  children,
+  isOpen: isOpenDefault = false,
+  onDismiss,
+  id = 'store-dropdown',
+}: DropdownProps) => {
+  const [isOpen, setIsOpen] = useState(false)
+  const dropdownItemsRef = useRef<HTMLButtonElement[]>([])
+  const selectedDropdownItemRef = useRef(0)
+  const buttonDropdownRef = useRef<HTMLButtonElement>(null)
```

Comment:
How about keeping the `dropdown` prefix in the beginning?

```suggestion
  const dropdownButtonRef = useRef<HTMLButtonElement>(null)
```